extensions/wikia/AdEngine/js/slot/slotTargeting.js

```diff
@@ -2,9 +2,10 @@
 define('ext.wikia.adEngine.slot.slotTargeting', [
 	'ext.wikia.adEngine.adContext',
 	'ext.wikia.adEngine.utils.math',
+	'ext.wikia.adEngine.wrappers.prebid',
 	'wikia.abTest',
 	'wikia.instantGlobals'
-], function (adContext, math, abTest, instantGlobals) {
+], function (adContext, math, prebid, abTest, instantGlobals) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)